js/src/ui/GasPriceEditor/store.js

```diff
@@ -104,6 +104,9 @@ export default class GasPriceEditor {
   }
 
   @action setConditionDateTime = (time) => {
+    time.setMilliseconds(0); // ignored by/not passed to Parity
```

Comment:
Verified that it doesn't affect the components used with actual testing prior to pushing. So not going to "fix" it.